finnemit/data/LCTFuelLoad_fuel4_revisit20190521.csv

```diff
@@ -0,0 +1,18 @@
+Code,TREE,HERB
```

Comment:
This file should be removed - we don't want to add another loading file -- instead you should modify the existing file.

Reply:
I was to keep the filename consistent with what Christine gave me...  File name is pretry much how she tracks version.  But i see your point as well...  The CSV file has no metadata or anything, file name is pretty much only way to guess where they come from...

Reply:
In that case, can you please edit the existing `land-cover-gm2.csv` file, and add a comment at the top of the file (lines beginning with `#`) that adds any necessary metadata? 

One advantage of using git for version control is that we can track exactly which values have changed easily.

Reply:
Ok @yosukefk - this PR is almost ready from my perspective. If you could delete this file (`finnemit/data/LCTFuelLoad_fuel4_revisit20190521.csv `), that would be great!